src/aiod/__init__.py

```diff
@@ -17,5 +16,3 @@
 from aiod.resources import publications
 from aiod.resources import services
 from aiod.resources import teams
-
-__version__ = version(__name__)
```

Comment:
This does require us to duplicate information. But commitizen is configured to bump it in both places at once. Alternatively we can make the `pyproject.toml` dynamic instead. But the problem with this method is that it does not seem to work any longer as it tries to look up the version for "aiod" while the package would be installed as "aiondemand".